src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -2252,6 +2253,24 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 break;
             }
 
+            case NI_Sve_LoadVectorFirstFaulting:
+            {
+                insScalableOpts sopt = INS_SCALABLE_OPTS_LSL_N;
+                if (opt == INS_OPTS_SCALABLE_B)
+                {
+                    sopt = INS_SCALABLE_OPTS_NONE;
+                }
+                GetEmitter()->emitIns_R_R_R_R(ins, emitSize, targetReg, op1Reg, op2Reg, REG_ZR, opt, sopt);
+                break;
+            }
+
+            case NI_Sve_SetFfr:
+            {
+                assert(targetReg == REG_NA);
```

Comment:
any reason why we need special codegen handling for this?

Reply:
I'll see if I can remove the special codegen.

Reply:
We need special codegen, because it tries to emit under `emitIns_R_R` when it can only do `emitIns_R`.